docs/README.md

```diff
@@ -44,6 +44,8 @@ The table below lists the Swift types and their corresponding C# types.
 | `Swift.UInt16`                  | `ushort` |
 | `Swift.Int8`                    | `sbyte`  |
 | `Swift.UInt8`                   | `byte`   |
+| `Swift.UnsafePointer`           | `void*`  |
```

Comment:
I know we touched on this before - it does not feel right to drop the type information for the strongly typed pointers. Strong typing is an extra level of protection against programming bugs that we are giving away here.

Reply:
Ok, as we discussed, a thin layer would be useful here.